lively.collab/comments/commentMorph.js

```diff
@@ -217,11 +217,38 @@ export class CommentMorph extends Morph {
 
     this.isInEditMode = false;
 
+    this.resetButtons();
+    this.ui.editSaveButton.tooltip = 'Edit Comment';
+
     this.ui.commentTextField.readOnly = true;
     this.ui.commentTextField.fill = Color.rgb(240, 243, 244);
     this.ui.commentTextField.borderStyle = 'none';
   }
 
+  resetButtons () {
+    const buttonColor = Color.rgb(127, 140, 141);
+
+    this.ui.resolveButton.fontColor = buttonColor;
+    this.ui.resolveButton.nativeCursor = 'pointer';
+    this.ui.resolveButton.tooltip = this.comment.isResolved() ? 'Unresolve Comment' : 'Resolve Comment';
+
+    this.ui.deleteButton.fontColor = buttonColor;
+    this.ui.deleteButton.nativeCursor = 'pointer';
+    this.ui.deleteButton.tooltip = 'Delete Comment';
+  }
+
+  deactivateButtons () {
+    const deactivatedButtonColor = Color.rgba(127, 140, 141, 0.22);
```

Comment:
```suggestion
    const deactivatedButtonColor = Color.rgba(127, 140, 141, 0.4);
```
Imo, the buttons were a bit too transparent with 0.22